tests/test_tv.py

```diff
@@ -0,0 +1,33 @@
+import torch
+import pytest
+
+from photosynthesis_metrics import total_variation, TVLoss
+
+
+@pytest.fixture(scope='module')
+def prediction() -> torch.Tensor:
+    return torch.rand(4, 3, 256, 256)
+
+
+@pytest.fixture(scope='module')
+def target() -> torch.Tensor:
+    return torch.rand(4, 3, 256, 256)
+
+
+# ================== Test function: `total_variation` ==================
+def test_tv_works(prediction: torch.Tensor) -> None:
+    measure = total_variation(prediction, size_average=True, reduction_type='l2')
+    assert measure > 0 
+    measure = total_variation(prediction, size_average=True, reduction_type='l1')
+    measure = total_variation(prediction, size_average=True, reduction_type='l2_squared')
+
+
+
+# ================== Test class: `TVLoss` ==================
+def test_tvloss_init(prediction: torch.Tensor, target: torch.Tensor) -> None:
+    l = TVLoss()
+
```

Comment:
Formatting: and extra blank line here.

From the Google style guide (section 3.5):
**Two blank lines between top-level definitions**, be they function or class definitions. One blank line between method definitions and between the class line and the first method. No blank line following a def line. Use single blank lines as you judge appropriate within functions or methods.